src/Applications/LDAS_App/ldas_setup

```diff
@@ -1419,7 +1446,11 @@ def _printExeInputKeys(rqdExeInpKeys):
     print '#  See README files in ./src/Applications/LDAS_App/doc     #'
     print '#                                                          #'
     print '############################################################'
+    print '# for adas-coupling exp ( COUPLING = 1) '
+    print '# BCS_PATH needs to be consistent with that is defined '
+    print '# in ${ADAS_exp}/run/lnbcs'
```

Comment:
move these help comments up to where COUPLING is defined

Reply:
moved the text to LADAS COUPLING EXP INFO in ldas_setup.